server/routes/topics.js

```diff
@@ -1,27 +1,17 @@
 const express = require('express');
-const { path } = require('ramda');
 const Sentry = require('@sentry/node');
 const { logger } = require('../utils/logger');
 
-const fixUrls = element => {
-  const { id, description, href, linkText } = element;
-  switch (element.href) {
-    // TODO: Re-enable when Incentives personalization feature released
-    // case '/content/4204':
-    //   return { id, description, href: '/incentives', linkText };
-    default:
-      return { id, description, href, linkText };
-  }
-};
-
 const createTopicsRouter = ({ topicsService }) => {
   const router = express.Router();
 
   router.get('/', async (req, res, next) => {
     try {
       const userName = req.user && req.user.getFullName();
-      const establishmentId = path(['session', 'establishmentId'], req);
-      const topics = await topicsService.getTopics(establishmentId);
+      const { establishmentId, establishmentName } = req.session;
+      const topics = await topicsService.getTopics(
+        establishmentId && establishmentName,
```

Comment:
maybe should fix at source - it seems establishment name is always set based on dodgy regex, we only know its a valid establishment if we can find the associated id.

Reply:
Fixed this in the middleware